lv_conf_template.h

```diff
@@ -38,6 +38,9 @@
     /*Size of the memory available for `lv_malloc()` in bytes (>= 2kB)*/
     #define LV_MEM_SIZE (128U * 1024U)          /*[bytes]*/
 
+    /*Size of the TLSF pool maximum size(must be greater than LV_MEM_SIZE)*/
+    #define LV_TLSF_MAX_POOL_SIZE (128U * 1024U) /*[bytes]*/
```

Comment:
We don't expose that we use TLSF internally. So it should be 
`LV_MEM_MAX_POOL_SIZE`.

Reply:
I think we can use the value of `LV_MEM_POOL_EXPAND_SIZE` as the expansion size of `LV_MEM_SIZE`. If the user does not need this function, set `LV_MEM_POOL_EXPAND_SIZE` to 0.